include/spdlog/details/periodic_worker.h

```diff
@@ -38,6 +38,7 @@ class SPDLOG_API periodic_worker {
             }
         });
     }
+    std::thread *get_thread() { return &worker_thread_; }
```

Comment:
why not return reference like in the rest of this pr?

Reply:
Fair question :) 
To be honest I was just lazy to change some legacy APIs in my code. But that's not a good enough reason and indeed it makes no sense. I'll update the PR.